src/lib/layout/unauthenticated.svelte

```diff
@@ -126,254 +55,10 @@
-            padding: pxToRem(18) pxToRem(28);
-            border: pxToRem(2) solid hsl(343 98% 60% / 0.2);
-            border-radius: pxToRem(16);
-            background: rgba(253, 54, 110, 0.1);
-            box-shadow: 0 -12.173px 20.289px 0px rgba(253, 54, 110, 0.08) inset;
-            text-align: center;
-            display: flex;
-            justify-content: center;
-            align-items: center;
-            &-letter {
-                width: 2rem;
-                @media #{$break1} {
-                    width: 1rem;
-                }
-            }
-            @media #{$break1} {
-                padding: pxToRem(8) pxToRem(12);
-                border-radius: pxToRem(8);
-            }
-        }
-    }
-
-    :global(.theme-dark) .appwrite-pro {
-        color: hsl(var(--color-neutral-10));
-    }
-
-    .now-available {
-        position: relative;
-        z-index: 1;
-        background: linear-gradient(70deg, #fb5491 -35.72%, #19191d 79.96%);
-        background-clip: text;
-        -webkit-background-clip: text;
-        -webkit-text-fill-color: transparent;
-        margin-inline: auto;
-
-        @media #{$break1} {
-            font-size: pxToRem(18);
-            margin-block-start: pxToRem(12);
-        }
-        @media #{$break2open} {
-            font-size: pxToRem(30);
-            margin-block-start: pxToRem(36);
-        }
-    }
-    :global(.theme-dark) .now-available {
-        background: linear-gradient(89deg, #fb5491 -29.25%, #fff 43.27%);
-        background-clip: text;
-        -webkit-background-clip: text;
-        -webkit-text-fill-color: transparent;
-    }
-
-    .side-bg {
-        position: relative;
-        background-color: #ededf0;
-        overflow: hidden;
-    }
-    .side-bg::before {
-        position: absolute;
-        inset-block-start: -950px;
-        inset-inline-end: -650px;
-        content: '';
-        display: block;
-        inline-size: 100%;
-        block-size: 100%;
-        background: radial-gradient(49.55% 43.54% at 47% 50.69%, #e7f8f7 0%, #85dbd8 100%);
-        filter: blur(250px);
-        @media #{$break1} {
-            inset-block-start: -200px;
-            inset-inline-end: -400px;
-            filter: blur(100px);
-        }
-    }
-    .side-bg::after {
-        position: absolute;
-        inset-block-end: -850px;
-        inset-inline-start: -600px;
-        content: '';
-        display: block;
-        inline-size: 100%;
-        block-size: 100%;
-        background: radial-gradient(50% 46.73% at 50% 53.27%, #fe9567 28.17%, #fd366e 59.38%);
-        filter: blur(250px);
-
-        @media #{$break1} {
-            inset-block-end: -200px;
-            inset-inline-start: -400px;
-            filter: blur(100px);
-        }
-    }
-    :global(.theme-dark) .side-bg {
-        background-color: #19191d;
-    }
-
-    /****** OLD ******/
     /* Default (including mobile) */
     #main section:first-child {
         padding-block-start: 2.25rem;
```

Comment:
can you check if these are needed?

Reply:
based on svelte diagnosis, i removed everything unused, i will double check and remove the `/****** OLD ******/` in the process.